### PR TITLE
i hecking love surf!

### PR DESCRIPTION
**Pull request by Whennya**
Add configs for 18 surf maps:
- ze_re_surf_skyline_ep_rf3
- ze_surf_bona_s2
- ze_surf_charge_v2_f2
- ze_surf_cherry_blossom_v4_1
- ze_surf_danger
- ze_surf_dark_fantasy_v1
- ze_surf_daybreak_v1_1
- ze_surf_facility_v2_3
- ze_surf_gypt_v1_3s
- ze_surf_hp_v1_2_s2
- ze_surf_last_point_v1_5
- ze_surf_luna_v3_f4
- ze_surf_outside_s3
- ze_surf_saltfish_css2
- ze_surf_shonyudo_v5_2s
- ze_surf_sky_v1_3
- ze_surf_sphinx_stronghold_a5
- ze_surf_virtus_s_ep_f4

### CODE REVIEW

File: musicname/ze_surf_bona_s2.cfg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"music"
+{
+    "kiss.mp3"                                              "BAMBOO BIMBO - KISS"
+    "nothings gonna stop us tonight.mp3"                    "ANNALISE - NOTHING GONNA STOP US TONIGHT"
```
Comment (notkoen) [inline musicname/ze_surf_bona_s2.cfg]:
```suggestion
    "kiss.mp3"                                              "Bamboo Bimbo - Kiss"
    "nothings gonna stop us tonight.mp3"                    "Annalise - Nothing's Gonna Stop Us Tonight"
```

File: musicname/ze_surf_dark_fantasy_v1.cfg [hunk 2]
```diff
@@ -0,0 +1,30 @@
+"music"
+{
+    "headhunterz feat. kshmr - dharma (extended mix).mp3"                       "Headhunterz & KSHMR - Dharma"
+    "julian jordan - pilot.mp3"                                                 "Julian Jordan - Pilot"
+    "mix_creepergirl.mp3"                                                       "Domastic - Khalas , Wubbaduck - Rush"
+    "muzzy - new age (feat. celldweller) [monstercat release].mp3"              "Muzzy - New Age (feat. Celldweller)"
```
Comment (notkoen) [inline musicname/ze_surf_dark_fantasy_v1.cfg]:
```suggestion
    "muzzy - new age (feat. celldweller) [monstercat release].mp3"              "Muzzy - New Age (ft. Celldweller)"
```

File: musicname/ze_surf_dark_fantasy_v1.cfg [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "red zone remix - error.mp3"                                                "Red Zone Remix - error"
+    "animal rights - rain 711.mp3"                                              "Animal Rights - Rain 7.11"
+    "droeloe feat. nevve - backbone.mp3"                                        "DROELOE - BACKBONE (feat. Nevve)"
```
Comment (notkoen) [inline musicname/ze_surf_dark_fantasy_v1.cfg]:
```suggestion
    "droeloe feat. nevve - backbone.mp3"                                        "DROELOE - BACKBONE (ft. Nevve)"
```

File: musicname/ze_surf_dark_fantasy_v1.cfg [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "ofdream - first woe.mp3"                                                   "Ofdream - First Woe"
+    "barren gates feat. m.i.m.e - enslaved.mp3"                                 "Barren Gates & M.I.M.E - Enslaved"
+    "barren gates feat. rmnd - in the darkness.mp3"                             "Barren Gates & RMND - In The Darkness"
+    "neovaii - heart shaped box.mp3"                                            "Neovaii - Heart Shaped Box"
+    "telykast feat. jordy - somebody else (kc audio remix).mp3"                 "TELYKast - Somebody Else (feat. JORDY, KC Audio Remix)"
+    "triple m - 2gether.mp3"                                                    "Triple M - 2GETHER"
+    "varien feat. laura brehm - valkyrie.mp3"                                   "Varien - Valkyrie (feat. Laura Brehm)"
```
Comment (notkoen) [inline musicname/ze_surf_dark_fantasy_v1.cfg]:
```suggestion
    "telykast feat. jordy - somebody else (kc audio remix).mp3"                 "TELYKast - Somebody Else (ft. JORDY, KC Audio Remix)"
    "triple m - 2gether.mp3"                                                    "Triple M - 2GETHER"
    "varien feat. laura brehm - valkyrie.mp3"                                   "Varien - Valkyrie (ft. Laura Brehm)"
```

File: musicname/ze_surf_dark_fantasy_v1.cfg [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    "jon bellion - all time low (boxinlion remix).mp3"                          "Jon Bellion - All Time Low (BOXINLION Remix)"
+    "niviro - flashes.mp3"                                                      "NIVIRO - Flashes"
+    "stellar - alexeps.mp3"                                                     "AlexEps - Stellar"
+    "alan walker - different world.mp3"                                         "Alan Walker - Different World feat. Sofia Carson, K-391 & CORSAK"
```
Comment (notkoen) [inline musicname/ze_surf_dark_fantasy_v1.cfg]:
```suggestion
    "alan walker - different world.mp3"                                         "Alan Walker - Different World (ft. Sofia Carson, K-391 & CORSAK)"
```

File: musicname/ze_surf_dark_fantasy_v1.cfg [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    "efisio cross - if you fall i will carry you.mp3"                           "Efisio Cross - If you fall i will carry you"
+    "umineko no naku koro ni - opening.mp3"                                     "Umineko no naku koro ni opening - Katayoku no Tori"
+    "end of silence - when will you save me (feat. julie elven).mp3"            "End Of Silence - When Will You Save Me (feat. Julie Elven)"
```
Comment (notkoen) [inline musicname/ze_surf_dark_fantasy_v1.cfg]:
```suggestion
    "end of silence - when will you save me (feat. julie elven).mp3"            "End Of Silence - When Will You Save Me (ft. Julie Elven)"
```

File: musicname/ze_surf_dark_fantasy_v1.cfg [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    "barren gates feat. rmnd - in the darkness.mp3"                             "Barren Gates & RMND - In The Darkness"
+    "neovaii - heart shaped box.mp3"                                            "Neovaii - Heart Shaped Box"
+    "telykast feat. jordy - somebody else (kc audio remix).mp3"                 "TELYKast - Somebody Else (feat. JORDY, KC Audio Remix)"
+    "triple m - 2gether.mp3"                                                    "Triple M - 2GETHER"
+    "varien feat. laura brehm - valkyrie.mp3"                                   "Varien - Valkyrie (feat. Laura Brehm)"
+    "jon bellion - all time low (boxinlion remix).mp3"                          "Jon Bellion - All Time Low (BOXINLION Remix)"
+    "niviro - flashes.mp3"                                                      "NIVIRO - Flashes"
+    "stellar - alexeps.mp3"                                                     "AlexEps - Stellar"
+    "alan walker - different world.mp3"                                         "Alan Walker - Different World feat. Sofia Carson, K-391 & CORSAK"
+    "efisio cross - if you fall i will carry you.mp3"                           "Efisio Cross - If you fall i will carry you"
```
Comment (notkoen) [inline musicname/ze_surf_dark_fantasy_v1.cfg]:
```suggestion
    "efisio cross - if you fall i will carry you.mp3"                           "Efisio Cross - If You Fall I Will Carry You"
```

File: musicname/ze_surf_last_point_v1_5.cfg
Comment (notkoen) [inline musicname/ze_surf_last_point_v1_5.cfg:1]: For the different music from Black Mesa, you could change it to `Joel Nielsen - <music name> (Black Mesa OST)`

**Review by Whennya — COMMENTED** [review]
updated

File: musicname/ze_surf_dark_fantasy_v1.cfg [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    "barren gates feat. m.i.m.e - enslaved.mp3"                                 "Barren Gates & M.I.M.E - Enslaved"
+    "barren gates feat. rmnd - in the darkness.mp3"                             "Barren Gates & RMND - In The Darkness"
+    "neovaii - heart shaped box.mp3"                                            "Neovaii - Heart Shaped Box"
+    "telykast feat. jordy - somebody else (kc audio remix).mp3"                 "TELYKast - Somebody Else (feat. JORDY, KC Audio Remix)"
+    "triple m - 2gether.mp3"                                                    "Triple M - 2GETHER"
+    "varien feat. laura brehm - valkyrie.mp3"                                   "Varien - Valkyrie (ft. Laura Brehm)"
+    "jon bellion - all time low (boxinlion remix).mp3"                          "Jon Bellion - All Time Low (BOXINLION Remix)"
+    "niviro - flashes.mp3"                                                      "NIVIRO - Flashes"
+    "stellar - alexeps.mp3"                                                     "AlexEps - Stellar"
+    "alan walker - different world.mp3"                                         "Alan Walker - Different World ft. (Sofia Carson, K-391 & CORSAK)"
```
Comment (notkoen) [inline musicname/ze_surf_dark_fantasy_v1.cfg]:
```suggestion
    "alan walker - different world.mp3"                                         "Alan Walker - Different World (ft. Sofia Carson, K-391 & CORSAK)"
```

File: musicname/ze_surf_dark_fantasy_v1.cfg [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    "mix_creepergirl.mp3"                                                       "Domastic - Khalas , Wubbaduck - Rush"
+    "muzzy - new age (feat. celldweller) [monstercat release].mp3"              "Muzzy - New Age (ft. Celldweller)"
+    "red zone remix - error.mp3"                                                "Red Zone Remix - error"
+    "animal rights - rain 711.mp3"                                              "Animal Rights - Rain 7.11"
+    "droeloe feat. nevve - backbone.mp3"                                        "DROELOE - BACKBONE (ft. Nevve)"
+    "ofdream - first woe.mp3"                                                   "Ofdream - First Woe"
+    "barren gates feat. m.i.m.e - enslaved.mp3"                                 "Barren Gates & M.I.M.E - Enslaved"
+    "barren gates feat. rmnd - in the darkness.mp3"                             "Barren Gates & RMND - In The Darkness"
+    "neovaii - heart shaped box.mp3"                                            "Neovaii - Heart Shaped Box"
+    "telykast feat. jordy - somebody else (kc audio remix).mp3"                 "TELYKast - Somebody Else (feat. JORDY, KC Audio Remix)"
```
Comment (notkoen) [inline musicname/ze_surf_dark_fantasy_v1.cfg]:
```suggestion
    "telykast feat. jordy - somebody else (kc audio remix).mp3"                 "TELYKast - Somebody Else (ft. JORDY, KC Audio Remix)"
```

**Review by notkoen — CHANGES_REQUESTED** [review]
Remaining issues I spotted. Hopefully I didn't mist anything.

**Pull request merged**